server/web/controller.go

```diff
@@ -722,3 +722,34 @@ func (c *Controller) XSRFFormHTML() string {
 func (c *Controller) GetControllerAndAction() (string, string) {
 	return c.controllerName, c.actionName
 }
+
+// JSONResp sends JSON response directly
+func (c *Controller) JSONResp(data interface{}, encoding ...bool) error {
```

Comment:
I think we don't need `encoding` params in most cases. So I think if users want to using this param, they could invoke `ServeJSON` manually.